Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
```

Comment:
You'll need to update the build scripts in release too.

Reply:
You nailed it! https://github.com/openshift/release/pull/9029